src/View.zig

```diff
@@ -0,0 +1,161 @@
+//! A View is effectively an "oversized" Window that can be written to and rendered in pieces.
+
+const std = @import("std");
+const mem = std.mem;
+
+const View = @This();
+
+const Screen = @import("Screen.zig");
+const Window = @import("Window.zig");
+const Unicode = @import("Unicode.zig");
+const Cell = @import("Cell.zig");
+
+/// View Allocator
+alloc: mem.Allocator,
+/// Underlying Screen
+screen: *Screen,
+/// Underlying Window
+win: *Window,
+
+pub const Config = struct {
+    max_width: usize = 10_000,
+    max_height: usize = 10_000,
+    x_pixel: usize = 0,
+    y_pixel: usize = 0,
```

Comment:
Pixels shouldn't matter in this view? They only come into play later in rendering. Let's take them out of the Config struct and set them to 0 in `init`

Reply:
Yeah, they would. The image is pretty much a flag on it's upper-left cell, and when that gets drawn to the *real* screen all of the data will be there to properly draw it.

Now...making a *scrollable* image would be much harder because you'd need to calculate it's extents and then use clip regions in the kitty protocol to get it to work. So you would need the pixel density to know how many cells the image takes up for this case.